frontend/public/locales/en/translation.json

```diff
@@ -1514,6 +1514,7 @@
   "Hosted control plane": "Hosted control plane",
   "Hosted control plane is already upgrade to the latest version available. Cluster node pools can be upgraded to match the control plane.": "Hosted control plane is already upgrade to the latest version available. Cluster node pools can be upgraded to match the control plane.",
```

Comment:
```suggestion
  "Hosted control plane is already upgraded to the latest version available. Cluster node pools can be upgraded to match the control plane.": "Hosted control plane is already upgraded to the latest version available. Cluster node pools can be upgraded to match the control plane.",
```

Reply:
I know this is not part of your work @vishsanghishetty, but it's a small thing we might as well fix now.